src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/config/CourtDetailsConfiguration.java

```diff
@@ -21,4 +23,15 @@ public CourtDetailsConfiguration(ObjectMapper objectMapper) throws IOException {
             });
         }
     }
+
+    public CourtDetailsTemplateFields buildCourtDetailsTemplateFields(String courtSelection) {
```

Comment:
Does this belong in this class? It seems irrelevant to court details configuration.